jetty-maven/jetty-maven-core/src/main/java/org/eclipse/jetty/maven/AbstractUnassembledWebAppMojo.java

```diff
@@ -0,0 +1,67 @@
+//
+// ========================================================================
+// Copyright (c) 1995-2022 Mort Bay Consulting Pty Ltd and others.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License v. 2.0 which is available at
+// https://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+// ========================================================================
+//
+
+package org.eclipse.jetty.maven;
+
+import java.io.File;
+
+import org.apache.maven.plugins.annotations.Parameter;
+
+/**
+ * Base class for all goals that operate on unassembled webapps.
+ *
+ */
+public abstract class AbstractUnassembledWebAppMojo extends AbstractWebAppMojo
```

Comment:
There's something not quite right with the inheritance hierarchy here. There's no point in having this class in jetty-maven-core if the only thing that uses it is in jetty-maven-plugin.

Reply:
https://github.com/eclipse/jetty.project/pull/9216#discussion_r1095253119

Reply:
https://github.com/eclipse/jetty.project/pull/9216#discussion_r1095263999

Reply:
class moved here 4435939a33b8f1b1dc4c5d1d350f13396b520a67